server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -595,6 +595,16 @@ public final class IndexSettings {
         Property.Dynamic
     );
 
+    public static final TimeValue DEFAULT_REMOTE_TRANSLOG_BUFFER_INTERVAL = new TimeValue(650, TimeUnit.MILLISECONDS);
```

Comment:
nit: `650` seems like a specific number. Why is it a default?

Reply:
This has been set to 650ms given the ongoing perf tests where the 650ms is giving higher throughput in comparison to other values of buffer interval tested. This is configurable and we can get it changed based on further testing discovery.